### PR TITLE
Portfolio setup and mobile first

### PR DESCRIPTION
**Pull request by VivekRaaZ**
In this project milestone :

1. Created an index.html file
2. Created a style.css file
3. Created a Readme file
4. Added Linters to the project

### CODE REVIEW

**Comment by VivekRaaZ** [comment]
README.md file is updated to good portfolio.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+</head>
+<body>
+  <main class="rj">
+    <header>
+      <div>      <a href="#" class="logo">My Logo</a></div>
+
+      <div>
+        <img src="image/Union.png" alt="button" />
+      
+    </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please wrap this hamburger button with a `nav` tag for semantics reason and the right position for convention and user experience for mobile devices. Please replace `div tag` with `nav tag`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    </header>
+    <section>
+      <h1>I'm Vivek Raj glad to see you !</h1>
+      <p class="software">
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please use `color: rgba(23, 43, 77, 1);` to your `h1` tag to stick to the design as much as possible.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        I’m a software developer! I can help you build a product, 
+        feature or website Look through some of my work and 
+        experience! If you like what you see 
+        and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible like color, spacing etc. I would suggest you please descrese the spacing between heading and section to stick to the design.
### Your design
his

File: index.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            </a>
+          </li>
+          <li>
+            <a href="https://www.angellist.com/">
+              <img src="image/angellist.png" alt="angellist-icon" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible like color, spacing, font, etc. I would suggest, please use the small background image as some are missing. Also place it at the proper place.
### Your design
hisssssss

### Figma design
his

File: index.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+      <h1>I'm Vivek Raj glad to see you !</h1>
+      <p class="software">
+        I’m a software developer! I can help you build a product, 
+        feature or website Look through some of my work and 
+        experience! If you like what you see 
+        and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+      <p class="connect">LET'S CONNECT</p>
+      <div class="media">
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I think it is better to use` hx` tag like `h3,h4,h5` tag for Let's connect as this is heading for social media icons.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @VivekRaaZ, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good design ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] You branch name is not bad but you can make it more descriptive like It would be better if it was ` Portfolio_setup` So please from next time use you branch name according to project. Like If the project is desktop version then you can name is `desktop` etc.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (VivekRaaZ) [reply index.html]: @roshan-bajgain
in my pc its working very well at width:375; and height:768;

i don't know it's behave like miss match.
Screenshot 2023-07-08 141037

File: index.html (hunk 4 above)
Comment (VivekRaaZ) [reply index.html]: @roshan-bajgain 
Screenshot 2023-07-08 141037

it's working very well, i think you change the right width and height then it mismatch the bubble.

Comment (roshan-bajgain) [reply index.html]: Hello @VivekRaaZ, I notice that in your section class, you have `height 78%` You can fix this issue by making it `100%` or `100vh` but you need to decrease the gap between the header and section of the page.
As it show only upto `78%` height that's why the background image is not showing. Also, I notice that the gap is have due to `margin-top: 110px;` please remove or decrease it according to the design.

### You CSS file
his

File: index.html (hunk 3 above)
Comment (roshan-bajgain) [reply index.html]: Please see the comment below and make the necessary changes. Thank you

File: index.html (hunk 2 above)
Comment (Nemwel-Boniface) [inline index.html]: - [x] There seems to be a slightly larger spacing between the header section and the heading section which is what @roshan-bajgain  mentioned [here](https://github.com/VivekRaaZ/portfolio/pull/1#discussion_r1257062807). Kindly reduce the paddings/ margins used to separate these two sections by a small amount to ensure that your design look more aesthetic by having a uniform design throughput.

**This is how the section looks like on my end**

image

**This is the desired design as provided in the Figma design:**

image

**Notice the slight design differences?**

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @VivekRaaZ  👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

I have received your second opinion request with a complaint about the previous reviewer @roshan-bajgain  Below are my findings:

I have seen that your complaints are of the design related requested changes. The reviewer was being very keen on the minor details of the design guidelines which in my opinion are good to ensure that your design looks as professional as possible. Remember this is the portfolio that you will be sharing for recruiters to see and the better it is the better for you to impress them. So about the following:
- [ ] I believe [this](https://github.com/VivekRaaZ/portfolio/pull/1#discussion_r1257067007) could be marked as [OPTONAL].
- [ ] Kindly implement any other change that was requested by @roshan-bajgain as they are very much correct and all help to improve the quality of your codebase by helping you make use of semantic tags and the correct tags for a specific section.

- [ ] Kindly check the inline comment left below.

In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

**Comment by VivekRaaZ** [comment]
I have done all changes according to code reviewer.
image

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @VivekRaaZ,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Nemwel-Boniface** [comment]
Good job @VivekRaaZ :clap:

**Comment by VivekRaaZ** [comment]
> Good job @VivekRaaZ 👏

Thank you Sir.

**Pull request merged**